extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php

```diff
@@ -9,7 +9,8 @@ class GlobalWatchlistBot {
 	const REPLY_ADDRESS = 'noreply@wikia.com';
 
 	public function __construct() {
-		\F::app()->wg->ExtensionMessagesFiles['GlobalWatchlist'] = dirname( __FILE__ ) . '/GlobalWatchlist.i18n.php';
+		$messageFiles = \F::app()->wg->ExtensionMessagesFiles;
+		$messageFiles['GlobalWatchlist'] = dirname( __FILE__ ) . '/GlobalWatchlist.i18n.php';
```

Comment:
Oh, hmm. Yea, when the bot is instantiated it's including the message files. Incorrectly at that. Looks like that was my change. Sorry about that. Thanks for fixing it.

But also, no, it doesn't really do anything. 1.) Because we're already including the file [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/GlobalWatchlist/GlobalWatchlist.php#L21) during startup. And 2.) because most of these messages are in the MediaWiki namespace on messaging wiki which trumps this file. I'm going to remove that include from the set-up file since these messages are only need when the bot is run.